shared/common-adapters/text.desktop.js

```diff
@@ -317,6 +318,11 @@ export const styles = {
     ...globalStyles.fontSemibold,
     ...sizeGroups['14']
   },
+  textBodySmallBold: {
```

Comment:
This is body small regular, white 40%, and white 75% on the username since it's an inline link.
i.e this style:
![screen shot 2016-05-25 at 2 31 31 pm](https://cloud.githubusercontent.com/assets/2807426/15551756/6513a5e8-2285-11e6-8777-45b6d88082a5.png)